INSTALL.md

```diff
@@ -854,6 +854,10 @@ This option will only produce a compliant FIPS provider if you have:
 
 Failure to do all of these will produce a non-compliant FIPS provider.
 
+### disable-fips-fcc-params
+
+Disable support for legacy FIPS186-4 FCC params.
+
```

Comment:
I think you mean `disable-fips-ffc-params` and FIPS 186-4 FFC params. However it still is fairly cryptic option name as someone could think of disabling all ffc support including the named primes.

Maybe `disable-legacy-fips-ffc` or `disable-fips186-4-ffc`?